ext/json/Makefile.frag.w32

```diff
@@ -1,4 +1,4 @@
-ext\json\json_scanner.c: ext\json\json_scanner.re
+ext\json\json_scanner.c ext\json\php_json_scanner_defs.h: ext\json\json_scanner.re
```

Comment:
`json_scanner.c` appears to depend on `json_parser.tab.h` as well:

```
; make ext/json/json_scanner.lo
/bin/bash /tmp/build/libtool --silent --preserve-dup-deps --tag CC --mode=compile cc -Iext/json/ -I/home/arnaud/dev/php-src/ext/json/ -I/tmp/build/include -I/tmp/build/main -I/home/arnaud/dev/php-src -I/usr/include/valgrind -I/tmp/build/ext/date/lib -I/home/arnaud/dev/php-src/ext/date/lib -I/tmp/build/TSRM -I/tmp/build/Zend -I/home/arnaud/dev/php-src/main -I/home/arnaud/dev/php-src/Zend -I/home/arnaud/dev/php-src/TSRM -I/tmp/build/    -fno-common -Wstrict-prototypes -Wformat-truncation -Wlogical-op -Wduplicated-cond -Wno-clobbered -Wall -Wextra -Wno-strict-aliasing -Wno-unused-parameter -Wno-sign-compare -g -O2 -fvisibility=hidden -Wimplicit-fallthrough=1 -DZEND_SIGNALS   -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -c /home/arnaud/dev/php-src/ext/json/json_scanner.c -o ext/json/json_scanner.lo  -MMD -MF ext/json/json_scanner.dep -MT ext/json/json_scanner.lo
/home/arnaud/dev/php-src/ext/json/json_scanner.c:22:10: fatal error: json_parser.tab.h: No such file or directory
   22 | #include "json_parser.tab.h"
      |          ^~~~~~~~~~~~~~~~~~~

```

Reply:
added